packages/dapp/src/pages/ViewInvoice.jsx

```diff
@@ -54,11 +54,9 @@ export const ViewInvoice = ({
     params: { hexChainId, invoiceId },
   },
 }) => {
-  const {
-    chainId,
-    account,
-    provider: ethersProvider,
-  } = useContext(Web3Context);
+  const { chainId, account, provider: ethersProvider } = useContext(
+    Web3Context,
+  );
```

Comment:
I think we might have some fighting linters going on. I think @psparacino formats this to the multiline format, and your's does it this way.